src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -439,6 +439,8 @@ Programs can only use <code>requires transitive java.base</code> when \
 preview features are enabled.<br>\
 Preview features may be removed in a future release, or upgraded \
 to permanent features of the Java Platform.<br>
+doclet.NonPreviewPreview=Programs can use {0} without enabling preview features. \
+    Programs can only reference the preview API by name when preview features are enabled.
```

Comment:
> Programs can only reference the preview API by name when preview features are enabled.

Should we reword like `Programs can only use {0} as {1} when preview features are enabled.` where `{1}` is (the list of) preview feature class(es) or interface(s) this existing class or interface extends/implements.